packages/fxa-customs-server/lib/server.js

```diff
@@ -105,6 +105,14 @@ module.exports = async function createServer(config, log) {
     host: config.listen.host,
   });
 
+  api.events.on({ name: 'request', channels: 'error' }, (request, event) => {
```

Comment:
It would be good to add the request context here. Sentry has a few utilities to extract it, I believe I used some of those in the auth-server to extract details from the request object.